openff/units/units.py

```diff
@@ -63,6 +68,19 @@ def _dask_finalize(results, func, args, units):
         values = func(results, *args)
         return Quantity(values, units)
 
+    @requires_package("openmm")
+    def to_openmm(self) -> "OpenMMQuantity":
+        """Convert the quantity to an `openmm.unit.Quantity`.
+
+        Returns
+        -------
+        openmm_unit : str
+            The OpenMM compatible unit.
```

Comment:
```suggestion
        """Convert the quantity to an ``openmm.unit.Quantity``.

        Returns
        -------
        openmm_quantity : openmm.unit.quantity.Quantity
            The OpenMM compatible quantity.
```

Sorry, I know getting the formatting of these things right is just a nightmare (because it took me 20 minutes to come up with these changes). I'll try and put a docstring style guide together when I get a chance but I'm kinda waiting to see if MyST will put together Markdown docstring support first - https://github.com/executablebooks/MyST-Parser/issues/228 - but accomplishing this is apparently extremely complex.

Reply:
Well, what I wrote up * is straight-up wrong no matter how it's formatted, so this suggested change is certainly welcome.

\* There's a decent chance it was Copilot's idea, but that doesn't excuse my lack of reviewing it